### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by JRahnama**

### CODE REVIEW

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommandBuilder.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  
+            ]]></format>
+        </remarks>
+    <related type="Article" href="~/docs/framework/data/adonet/retrieving-and-modifying-data.md">Retrieving and Modifying Data in ADO.NET</related>
+    <related type="Article" href="~/docs/framework/data/adonet/sql/index.md">SQL Server and ADO.NET</related>
+    <related type="Article" href="~/docs/framework/data/adonet/ado-net-overview.md">ADO.NET Overview</related>
+        </SqlCommandBuilder>
+    </members>
+</docs>
+</xml>
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommandBuilder.xml]: Nit: Please also add EOFs in Xmls

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                while (reader.Read())
+                {
+                    Console.WriteLine(String.Format("{0}, {1}",
+                        reader[0], reader[1]));
+                }
+            }
+        }
+        // </Snippet1>
+    }
+}
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs]: EOFs

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        static void Main()
+        {
+            string str = "Data Source=(local);Initial Catalog=Northwind;"
+                + "Integrated Security=SSPI";
+            string qs = "SELECT OrderID, CustomerID FROM dbo.Orders;";
+            CreateCommand(qs, str);
+
+        }
+        // <Snippet1>
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs]: Please move snippet xml tags to top

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommand.SqlCommand2.cs]: Is this file supposed to be here?

File: doc/samples/SqlParameterCollection_Add.cs [hunk 5]
```diff
@@ -10,4 +10,4 @@ public void AddSqlParameter(SqlCommand command)
         command.Parameters.Add(new SqlParameter("Description", "Beverages"));
     }
 }
-// </Snippet1>
+// </Snippet1>
```
Comment (cheenamalhotra) [inline doc/samples/SqlParameterCollection_Add.cs]: Why removing EOF, accidental changes?

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs [hunk 6]
```diff
@@ -2475,7 +2475,7 @@ public static void ChangePassword(string connectionString, string newPassword)
             }
         }
 
-        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlConnection.xml' path='docs/members[@name="SqlConnection"]/ChangePasswordConnectionStringCredentialNewPassworNewSecurePassword/*' />
+        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlConnection.xml' path='xml/docs/members[@name="SqlConnection"]/ChangePasswordConnectionStringCredentialNewPassworNewSecurePassword/*' />
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs]: Accidental changes?

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs [hunk 7]
```diff
@@ -2758,6 +2758,7 @@ internal byte[] GetBytes(object o, out Format format, out int maxSize)
     // You must not change the guid for this coclass
     // or the iid for the ISQLDebug interface
     //
+    /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDebugging.xml' path='docs/members[@name="SQLDebugging"]/SQLDebugging/*'/>
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs]: Also here, is this necessary?

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs [hunk 8]
```diff
@@ -4317,13 +4399,7 @@ private SqlParameter GetSqlParameterWithQueryText(string queryText)
...
         }
 
-        /// <summary>
-        /// Constructs the sp_describe_parameter_encryption request with the values from the original RPC call.
-        /// Prototype for <sp_describe_parameter_encryption> is 
-        /// exec sp_describe_parameter_encryption @tsql=N'[SQL Statement]', @params=N'@p1 varbinary(256)'
-        /// </summary>
-        /// <param name="originalRpcRequest">Original RPC request</param>
-        /// <param name="describeParameterEncryptionRequest">sp_describe_parameter_encryption request being built</param>
+
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs]: Plz don't delete any documentation, this is private method, not harming anything.

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml [hunk 9]
```diff
@@ -0,0 +1,1900 @@
...
+                    
+                    The `callback` parameter lets you specify an <xref:System.AsyncCallback> delegate that is called when the statement has completed. You can call the <xref:Microsoft.Data.SqlClient.SqlCommand.EndExecuteReader%2A> meth
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml]: Please change this to `Microsoft.Data.Sql.SqlNotificationRequest`

File: doc/snippets/Microsoft.Data.SqlClient/SqlCommandBuilder.xml [hunk 10]
```diff
@@ -0,0 +1,37 @@
+<xml version="1.0">
+<docs>
+    <members name="SqlCommandBuilder">
+        <SqlCommandBuilder>
+            <summary>Automatically generates single-table commands that are used to reconcile changes made to a <see cref="T:System.Data.DataSet" /> with the associated SQL Server database. This class cannot be inherited.</summary>
+            <remarks>
+                <format type="text/markdown"><![CDATA[  
+  
+                    ## Remarks  
+                    The <xref:System.Data.SqlClient.SqlDataAdapter> does not automatically generate the Transact-SQL statements required to reconcile changes made to a <xref:System.Data.DataSet> with the associated instance of SQL Server. However, you can create a <xref:System.Data.SqlClient.SqlCommandBuilder> object to automatically generate Transact-SQL statements for single-table updates if you set the <xref:System.Data.SqlClient.SqlDataAdapter.SelectCommand%2A> property of the <xref:System.Data.SqlClient.SqlDataAdapter>. Then, any additional Transact-SQL statements that you do not set are generated by the <xref:System.Data.SqlClient.SqlCommandBuilder>.
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlCommandBuilder.xml]: `Microsoft.Data.SqlClient` everywhere

File: doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopyOptions.xml [hunk 11]
```diff
@@ -0,0 +1,66 @@
+<docs>
+    <members name="SqlBulkCopyOptions">
+        <SqlBulkCopyOptions>
+            <summary>Bitwise flag that specifies one or more options to use with an instance of <see cref="T:System.Data.SqlClient.SqlBulkCopy" />.</summary>
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopyOptions.xml]: `Microsoft.Data.SqlClient`

File: doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+            </remarks>
+        </CreateCommand>
+        <CreateCommandBuilder>
+            <summary>Returns a strongly typed <see cref="T:System.Data.Common.DbCommandBuilder" /> instance.</summary>
+            <returns>A new strongly typed instance of <see cref="T:System.Data.Common.DbCommandBuilder" />.</returns>
+            <remarks>
+                <format type="text/markdown"><![CDATA[  
+  
+                    ## Examples  
+                    The following code fragment returns a strongly typed <xref:System.Data.Common.DbCommandBuilder> instance:
```
Comment (cheenamalhotra) [inline doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml]: This is also `Microsoft.Data.Common.DbCommandBuilder`

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs (hunk 6 above)
Comment (JRahnama) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs]: VS2019 throw some errors regarding file not find in the XML path. When I added the the full path of the tag in the xml VS did not complain.

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs (hunk 8 above)
Comment (JRahnama) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs]: probably we will end up having to summaries and params if the compiler does not complain about it. we already have params in the include file

**Pull request merged**